logger/index.ts

```diff
@@ -0,0 +1,15 @@
+import * as Sentry from '@sentry/react-native';
+
+export const createLogMsg = (level: Sentry.Severity) => (category: string, message: string) => {
```

Comment:
I like it :+1: 

whether it makes sense I'd define a new `Categiory` type like this:

```
type Category = 'app-storage' | 'BlueElectrum' | ...
``` 

to avoid potential typos